### ISSUE TITLE
Check for Girder and Romanesco

### ISSUE THREAD

**Issue by lukejharmon** [arborworkflows/ArborWebApps#135]
Upon restart I frequently (maybe always?) have to vagrant ssh and start romanesco, girder, or both. From the perspective of the web-app everything looks fine but nothing works. Is it possible to run some sort of check to see if these services are running and then return an informative error message? Otherwise end users will sometimes see what looks like a working Arbor but actually it is not - and it might make things easier for us to diagnose downstream.

**Comment by curtislisle** [comment]
Agreed.  I usually have to restart the services when I boot up.  We have had trouble getting upstart to start the daemons successfully on restarts.  I agree that it would be good to have a visual indication for users that services need to be restarted.  Thinking for just a second, I am not sure how to accomplish this offhand.  Glad we have an issue about it here.
